### ISSUE TITLE
Compiler FE Test: one-quantize core dump

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#4200]
Given invalid `input_data` aborted with core dump
```
$ one-quantize \
--input_dtype float32 \
--quantized_dtype uint8 \
--input_path ./inception_v3.circle \
--input_data ./inception_v3.circle \
--output_path ./inception_v3.quantized.circle
```
Problem: we should show error not abort with core dump

Source: SE with version 1.9.0

**Comment by jinevening** [comment]
Reason for the core dump.

When non-H5 file is given as an input_data, `record-minmax` throws `H5::FileIException`. The current implmentation of `safemain` only catches std exception, so other types of exceptions are not handled properly.

This problem was fixed with #4197 , which throws an std exception instead of `H5::FileIException` when non-H5 file is given (Thanks to @llFreetimell ).

**Comment by seanshpark** [comment]
done
